pom.xml

```diff
@@ -22,34 +22,45 @@
 
     <!-- Properties -->
     <properties>
-        <!-- Java target is 1.7 -->
+        <!-- Java target is 1.8 -->
         <!-- Make sure to adjust jdk version specific profiles when bumping this version -->
         <maven.compiler.target>1.8</maven.compiler.target>
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.argument.target>1.8</maven.compiler.argument.target>
         <maven.compiler.argument.source>1.8</maven.compiler.argument.source>
 
         <!-- Versioning -->
-        <version.org.apache.maven>3.9.3</version.org.apache.maven>
-
-        <version.org.apache.maven.plugins_maven-site-plugin>3.5.1</version.org.apache.maven.plugins_maven-site-plugin>
+        <version.com.google.inject.guice>5.1.0</version.com.google.inject.guice>
+        <version.commons-codec.commons-codec>1.15</version.commons-codec.commons-codec>
+        <version.commons-io.commons-io>2.13.0</version.commons-io.commons-io>
+        <version.eu.maveniverse.maven.mima>2.3.2</version.eu.maveniverse.maven.mima>
+        <version.org.apache.maven.maven>3.9.3</version.org.apache.maven.maven>
+        <version.org.apache.maven.plugin-tools.maven-plugin-annotations>3.5</version.org.apache.maven.plugin-tools.maven-plugin-annotations>
+        <version.org.apache.maven.shared.maven-invoker>3.0.0</version.org.apache.maven.shared.maven-invoker>
+        <version.org.arquillian.spacelift.arquillian-spacelift>1.0.2</version.org.arquillian.spacelift.arquillian-spacelift>
+        <version.org.codehaus.plexus.plexus-classworlds>2.7.0</version.org.codehaus.plexus.plexus-classworlds>
+        <version.org.codehaus.plexus.plexus-compiler-javac>2.7</version.org.codehaus.plexus.plexus-compiler-javac>
+        <version.org.codehaus.plexus.plexus-component-annotations>2.1.0</version.org.codehaus.plexus.plexus-component-annotations>
+        <version.org.codehaus.plexus.plexus-sec-dispatcher>2.0</version.org.codehaus.plexus.plexus-sec-dispatcher>
```

Comment:
Try to align versions where you can from the same projects.

Reply:
The goal of this issue is to clean up the poms, updating versions will be part of the future development